### PR TITLE
Hw 03 part fifth servlets

### PR DESCRIPTION
**Pull request by Alex-Korzun**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,13 +7,21 @@
...
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.22</version>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>4.0.1</version>
+            <scope>provided</scope>
+        </dependency>
+
```
Comment (Likh-Alex) [inline pom.xml]: redundant empty line)

**Review by Likh-Alex — APPROVED** [review]
Good)

File: pom.xml (hunk 1 above)
Comment (Alex-Korzun) [reply pom.xml]: yeah... here it is redundant I think)

Comment (dmytro-hryhoruk) [inline pom.xml]:
```suggestion
```

File: pom.xml [hunk 2]
```diff
@@ -54,14 +62,21 @@
...
+                    <artifactId>maven-war-plugin</artifactId>
+                    <configuration>
+                        <webXml>src/main/webapp/web.xml</webXml>
+                    </configuration>
+                </plugin>
             </plugins>
         </pluginManagement>
     </build>
-</project>
+</project>
```
Comment (dmytro-hryhoruk) [inline pom.xml]: add empty to the end of file(single line)

**Review by dmytro-hryhoruk — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (Alex-Korzun) [reply pom.xml]: done

File: pom.xml (hunk 2 above)
Comment (Alex-Korzun) [reply pom.xml]: done

**Pull request merged**